<?xml version="1.0"?>
<launch>
  <!-- 左右手参数 -->
  <arg name="left_id"    default="1"/>
  <arg name="right_id"   default="1"/>
  <arg name="left_port"  default="/dev/ttyUSB2"/>
  <arg name="right_port" default="/dev/ttyUSB1"/>
  <arg name="baud"       default="115200"/>
  <arg name="test_flag"  default="1"/>
  <arg name="serial_flag" default="1"/>

  <!-- 左手节点 -->
  <group ns="lefthand">
    <node name="inspire_hand" pkg="inspire_hand" type="inspire_hand" output="screen">
      <param name="hand_id" value="$(arg left_id)"/>
      <param name="portname" value="$(arg left_port)"/>
      <param name="baudrate" value="$(arg baud)"/>
      <param name="test_flags" value="$(arg test_flag)"/>
      <param name="Serial_flags" value="$(arg serial_flag)"/>
    </node>
  </group>

  <!-- 右手节点 -->
  <group ns="righthand">
    <node name="inspire_hand" pkg="inspire_hand" type="inspire_hand" output="screen">
      <param name="hand_id" value="$(arg right_id)"/>
      <param name="portname" value="$(arg right_port)"/>
      <param name="baudrate" value="$(arg baud)"/>
      <param name="test_flags" value="$(arg test_flag)"/>
      <param name="Serial_flags" value="$(arg serial_flag)"/>
    </node>
  </group>
</launch>
